src/Components/TodoList/TodoList.tsx

```diff
@@ -1,13 +1,19 @@
 import React from "react";
-import ListItem from "./ListItem/ListItem";
-import { Box } from "@mui/material";
+import ListItem, { ListItemProps } from "./ListItem/ListItem";
+import { Box, Divider } from "@mui/material";
 
-export default function TodoList() {
+interface TodoListProps {
+  items: ListItemProps[];
```

Comment:
You shouldn't use the prop of another component here
how about you create a types.ts file and make an interface for a task there, then all of the app component will work with it.